model/registry/geo.yaml

```diff
@@ -0,0 +1,113 @@
+groups:
+  - id: registry.geo
+    type: attribute_group
+    prefix: geo
+    brief: >
+      Geo fields can carry data about a specific location related to an event.
+      This geolocation information can be derived from techniques such as Geo IP, or be user-supplied.
+    attributes:
+      - id: city.name
```

Comment:
would we have a dedicated field for village/town/county name? 

Can we look into popular Maps API and see how this and other geo attributes would align with their abstractions?

E.g. [Google Maps](https://developers.google.com/maps/documentation/geocoding/requests-geocoding) or [Azure Maps](https://learn.microsoft.com/en-us/rest/api/maps/search/get-search-address?view=rest-maps-1.0&tabs=HTTP#geographicentitytype)

Reply:
let's either remove the attribute for now or consider more general term. Azure maps use `municipality`, google uses `locality`.

ChatGPT also prefers `locality` 

> Yes, the official term for the name of a city, town, village, or similar populated place is often referred to as a "locality" or "place name." In some contexts, it's also called a "geographic name" or "toponym."
>
> If you’re working with geographic data, “locality” is commonly used to encompass all such divisions, but “place name” is frequently used in casual contexts.